chapters/refereecommands.adoc

```diff
@@ -53,13 +64,15 @@ When the ball goes <<Ball In And Out Of Play, out of play>>, the following rules
 . The ball must be visible and must not be inside a field corner, a goal corner or behind the goal, before the ball placement starts
 . The <<Referee, referee>> can decide to place the ball manually at any time
 . The <<Referee, referee>> can decide to disable automatic ball placement for the rest of the game. TC/OC must agree with this decision
-. When a team has failed to place the ball 5 times in a row, it is not allowed to place the ball for the rest of the game half. All free kicks that were a result of the ball leaving the field, are awarded to the opposing team. For all other rule violations or when both teams failed to place the ball, the ball is placed by the <<Referee, referee>>
+. When a teams placement failure counter reached 5, it is not allowed to place the ball for the rest of the game half. All free kicks that were a result of the ball leaving the field, are awarded to the opposing team. For all other rule violations or when both teams failed to place the ball, the ball is placed by the <<Referee, referee>>
 . If no team can place the ball, the ball is placed by the <<Referee, referee>> or the <<Assistant Referee, assistant referee>>. Both the referee as well as the assistant referee are advised to use a so-called ball handler (a long, preferably black stick-like device) to move the ball.
 
-NOTE: If the ball is already at the placement position (for example after a failed free kick by the other team), the ball placement can be successful without a robot manipulating the ball. This could lead to the counter for failed attempts resetting without a contribution by the placing team.
-
 NOTE: The ball may still be moving when the placement command is issued.
 
+NOTE: If the ball is placed near to the defense area, teams are advised to keep the defense as close as possible to their target position.
```

Comment:
Firstly: I totally don't get this note.
Secondly: is "defense" a technical term that is well defined? Or is it just some robots that seem to be defending something?
Thirdly: What is the "target position"?

Reply:
Formulation adapted